web_app/dist/main.css

```diff
@@ -0,0 +1,159 @@
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 18px;
+    height: 100%;
+    background: #93ca68;
+    background-image: url(images/bg_logo.png);
+    background-repeat: repeat;   
+}   
+
+#content {
+    background: #fff;
+    color: #545454;
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 60%;
+    height: 90%;
+    margin-top: 50px;
+    padding-bottom: 20px;
+    border-radius: 10px;
+    box-shadow: 5px 25px 20px -10px #606060; 
+}
+
+#sticky-header {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
+
+/* The navigation bar */
+.navbar {
+    display: flex;
+    overflow: hidden;
+    background-color: #333;
+    width: 100%; /* Full width */
+    box-shadow: -5px 20px 50px -10px #656565; 
+}
+
+/* Style the search box inside the navigation bar */
+#search-bar {
+    display: flex;
```

Comment:
Is this necessary?? I don't think `display: flex` does anything if elements don't contain other elements

Reply:
Resolved, this diff thinks this is from a time where this line did nothing